### PR TITLE
Adding personal rep name to confirmation

### PR DESCRIPTION
**Pull request by adamconder**
Screenshot 2020-08-06 at 12 35 05
Screenshot 2020-08-06 at 12 35 01

### CODE REVIEW

File: app/controllers/confirmation/ConfirmationController.scala [hunk 1]
```diff
@@ -29,19 +33,43 @@ import scala.concurrent.ExecutionContext
...
-                                        config: FrontendAppConfig
+                                        config: FrontendAppConfig,
+                                        estatesConnector: EstatesConnector
                                       )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
+  private def personalRepName(personalRepresentative: PersonalRepresentativeType)
+                             (implicit request: Request[_]): String = {
+    personalRepresentative match {
+      case PersonalRepresentativeType(Some(EstatePerRepIndType(name, _, _, _, _, _, _, _)), None) =>
+        s"${name.firstName} ${name.lastName}"
```
Comment (jackheslop96) [inline app/controllers/confirmation/ConfirmationController.scala:48]: Could this be added as a def in the NameType case class?

File: app/controllers/confirmation/ConfirmationController.scala [hunk 2]
```diff
@@ -29,19 +33,43 @@ import scala.concurrent.ExecutionContext
...
+      case PersonalRepresentativeType(None, Some(EstatePerRepOrgType(name, _, _, _, _, _, _))) =>
+        name
+      case _ =>
+        val lang = request.lang
+        messagesApi("confirmationPage.personalRepresentative.default")(lang)
```
Comment (jackheslop96) [inline app/controllers/confirmation/ConfirmationController.scala:53]: I have a question about this sort of thing actually - is it good practice to declare vals for everything or is something like this a waste of memory?

Comment (adamconder) [reply app/controllers/confirmation/ConfirmationController.scala:53]: It's a waste of memory but I'm unsure when garbage collection would occur on the values.

Comment (adamconder) [reply app/controllers/confirmation/ConfirmationController.scala:53]: @jackheslop96 https://stackoverflow.com/questions/4138200/garbage-collection-on-a-local-variable

**Pull request merged**